app/src/main/java/org/luxoft/sdl_core/BleReader.java

```diff
@@ -0,0 +1,15 @@
+package org.luxoft.sdl_core;
+
+public abstract class BleReader {
```

Comment:
@ychernysheva please read about https://www.w3schools.com/java/java_interface.asp
I think in this specific case it is more preferrable than abstract class

Reply:
Done in 8effe4942498195cce7a335540df658a92a3e296